electrumx/server/block_processor.py

```diff
@@ -2687,6 +2753,9 @@ def advance_txs(
                 if atomical_id_of_distmint:
                     dft_count += 1
                     already_found_valid_operation = True
+                    # found vaild op
+                    self.logger.debug(f"mint ft {hash_to_hex_str(tx_hash)}")
+                    self.put_op_data(tx_num, "mint")
```

Comment:
I think we should name this 'mint-dft' to distinguish minting FT in decentralized mode.

We can also add 'dmint' for minting of NFT mints.

What do you think?